app/screens/content-view-screen/content-view-screen.tsx

```diff
@@ -42,8 +55,17 @@ export interface ContentViewNavigationStateParams {
 }
 export interface ContentViewScreenProps extends NavigationScreenProps<ContentViewNavigationStateParams> {}
 
+const injectedJavaScript = `${
+  /* Forward any post messages to WebView */ ""
+  }window.addEventListener('message', function(event) {
+    window.ReactNativeWebView.postMessage(JSON.stringify(event.data));
```

Comment:
Will website that contains CSP break this?

Reply:
Tested with `Content-Security-Policy: default-src 'self' button.like.co;` and it's working